src/index.html

```diff
@@ -17,6 +30,38 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="logo"
+        class="image"
+      />
+      <nav class="flex-text">
+        <ul>
+          <li>
+            <a
+              href="Apple"
```

Comment:
The `href` attribute for the 'Apple' link should be a valid URL. Consider using a relative or absolute URL, such as `href='/apple'` or `href='https://example.com/apple'`.